[{"\u00e9quipe": "040", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 74, in <module>\n    main()\n  File \"projet1.py\", line 40, in main\n    liste.append((d_debut, response['Time Series (Daily)'][d_debut][dic[mode]]))\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "042", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> usage: projet1.py [-h] [-d DATE] [-f DATE]\n                  [-v,--valeur {fermeture,ouverture,min,max,volume}]\n                  symbole [symbole ...]\nprojet1.py: error: unrecognized arguments: -v=volume\n</pre>\n"}, {"\u00e9quipe": "043", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> usage: projet1.py [-h] [-start DATE] [-end DATE]\n                  [-value {fermeture, ouverture, min, max, volume}]\n                  symbole [symbole ...]\nprojet1.py: error: unrecognized arguments: -v=volume -d=2018-09-21 -f=2018-09-24\n</pre>\n"}, {"\u00e9quipe": "044", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"}, {"\u00e9quipe": "045", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 42, in <module>\n    nouveau_dico = response['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "046", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"}, {"\u00e9quipe": "047", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 94, in <module>\n    donn\u00e9es(DEBUT, FIN)\n  File \"projet1.py\", line 80, in donn\u00e9es\n    don = RESPONSE['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "048", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 20, in <module>\n    print('{}({}, {}, {}'.format(args.symboles, args.v, args.f, args.d))\nAttributeError: 'Namespace' object has no attribute 'v'\n</pre>\n"}, {"\u00e9quipe": "049", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 105, in <module>\n    calcul_boursier(ARGS.symbole)\n  File \"projet1.py\", line 98, in calcul_boursier\n    tuple1 = (str(DEBUT), response['Time Series (Daily)'][str(DEBUT)][str(variable)])\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "050", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 83, in <module>\n    response['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "051", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Seulement 5 titres boursier peuvent \u00ebtre analys\u00e9s par minute\n</pre>\n"}, {"\u00e9quipe": "052", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 111, in <module>\n    affichage_final(ARGS.symbole, ARGS.d\u00e9but, ARGS.fin, ARGS.valeur)\n  File \"projet1.py\", line 104, in affichage_final\n    afficher_ieme_symbole(symbole, debut, fin, valeur)\n  File \"projet1.py\", line 90, in afficher_ieme_symbole\n    for key, value in dict_titre_boursier(debut, symbole)[\"Time Series (Daily)\"].items():\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "053", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 79, in <module>\n    display()\n  File \"projet1.py\", line 17, in display\n    print(request_data_alphavantage(symbole))\n  File \"projet1.py\", line 40, in request_data_alphavantage\n    for date in response['Time Series (Daily)']:\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "054", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 125, in <module>\n    main()\n  File \"projet1.py\", line 120, in main\n    res = traitment_donnee(req, arg)\n  File \"projet1.py\", line 90, in traitment_donnee\n    for cle_date_requete in reponse_api['Time Series (Daily)']:\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "055", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 86, in <module>\n    affichage(donnees(DEBUT, FIN, b), DATES, VALEUR)\n  File \"projet1.py\", line 71, in affichage\n    if dico['Time Series (Daily)'].get(i) is not None:\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "056", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"}, {"\u00e9quipe": "057", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"}, {"\u00e9quipe": "058", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"}, {"\u00e9quipe": "059", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"Projet1.py\", line 106, in <module>\n    afficherfinal(ARGS.symbole, ARGS.valeur, ARGS.d\u00e9but, ARGS.fin)\n  File \"Projet1.py\", line 99, in afficherfinal\n    afficherlisteval(symbole, valeur, debut, fin)\n  File \"Projet1.py\", line 85, in afficherlisteval\n    for i, value in dictvaleurs(symbole, 'compact')['Time Series (Daily)'].items():\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "060", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"}, {"\u00e9quipe": "061", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre>   File \"projet1.py\", line 34\n    import argparse\n         ^\nIndentationError: expected an indented block\n</pre>\n"}, {"\u00e9quipe": "062", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"Projet1.py\", line 124, in <module>\n    sortie(d\u00e9but, fin, alphavantage(x))\n  File \"Projet1.py\", line 112, in sortie\n    if str(ddate) in response['Time Series (Daily)'].keys():\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "063", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"}, {"\u00e9quipe": "064", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"Projet1.py\", line 75, in <module>\n    affichage(ARGS.symbole, ARGS.valeur, ARGS.d\u00e9but, ARGS.fin)\n  File \"Projet1.py\", line 69, in affichage\n    liste = list(tuple((datedebut, response['Time Series (Daily)'][datedebut][vd])))\nKeyError: 'Time Series (Daily)'\n</pre>\n"}, {"\u00e9quipe": "066", "score": 0, "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 100, in <module>\n    main()\n  File \"projet1.py\", line 51, in main\n    data['Time Series (Daily)'],\nKeyError: 'Time Series (Daily)'\n</pre>\n"}]